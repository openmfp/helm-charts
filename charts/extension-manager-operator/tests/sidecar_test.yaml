suite: platform-mesh istio sidecar
templates:
  - sidecar.yaml
tests:
- it: match the snapshot
  values:
    - ../values.yaml
    - ../test-values.yaml
  asserts:
    - matchSnapshot: { }
