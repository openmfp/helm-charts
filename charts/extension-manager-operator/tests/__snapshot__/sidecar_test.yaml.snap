match the snapshot:
  1: |
    apiVersion: networking.istio.io/v1beta1
    kind: Sidecar
    metadata:
      name: extension-content-operator
      namespace: NAMESPACE
    spec:
      outboundTrafficPolicy:
        mode: ALLOW_ANY
      workloadSelector:
        labels:
          app: RELEASE-NAME-extension-manager-operator
